### ISSUE TITLE
Make archival copies of old BOSC pages

### ISSUE THREAD

**Issue by nlharris** [OBF/bosc_materials#29]
especially from the wiki: http://obf.io/

At the least, we could download these (as html + images and/or PDFs) and save in this repo. Someday the wiki will stop working (whether planned or unplanned) and we don't want to lose all that historical information.

**Comment by hlapp** [comment]
I think you mean this as the wiki: https://www.open-bio.org/wiki/

The BOSC pages are here: https://www.open-bio.org/wiki/Category:BOSC_Conferences

Note that the "BOSC pages" prior to 2006 are actually just stubs for the actual BOSC websites, which were static websites and remain archived as such:
* https://open-bio.org/bosc2000/
* https://open-bio.org/bosc2001/
* https://open-bio.org/bosc2002/
* https://open-bio.org/bosc2003/
* https://open-bio.org/bosc2004/
* https://open-bio.org/bosc2005/

Note that none of these is tied to a wiki. If we wanted to archive these for perpetuity, I'd argue we should put them on Zenodo (even if they wouldn't be served as HTML sites from there, but presumably that's not the objective here?)

There are directories for later BOSCs, but these are only for some static files (such as the BOSC program). Again, to archive the digital content for perpetuity, I'd argue we should put them on Zenodo.

**Comment by nlharris** [comment]
Sure, if someone wants to do that that would be fine with me, though I'm unclear on exactly what we'd put on Zenodo. A PDF?

**Comment by hlapp** [comment]
> I'm unclear on exactly what we'd put on Zenodo.

Presumably, just the static site content. (We can create a Zip file and deposit that as well.) Not sure what a conversion to PDF would add.

**Comment by nlharris** [comment]
Right, so where I'm confused is how you'd put HTML + image files on Zenodo without using a zip file or something similar.

**Comment by hlapp** [comment]
You mean how would I upload more than one file to a Zenodo deposit (which is possible), or how do I upload HTML files to Zenodo so that Zenodo will serve them as HTML files (which is not possible)?

**Comment by shivprime94** [comment]
hey @nlharris, I can help you to upload the zip file on zonodo! Please let me know your availability so I can help you to set up the zonodo

**Comment by nlharris** [comment]
Thank you, @shivprime94! The hard part is not uploading the zip file, it's deciding what to zip and how to organize it. I'll let you know if there's a way for you to help! Your community spirit is much appreciated!

**Comment by akshit5565** [comment]
@hlapp  can you please assign me this task?

**Comment by akshit5565** [comment]
@nlharris can you assign me this task?

**Comment by nlharris** [comment]
@akshit5565 and @shivprime94 thank you for your kind offer, but this needs to be done by someone within our organization (which is why I removed the "help wanted" label just now--it was intended to mean "help wanted from someone else on the OBF Board").

**Comment by akshit5565** [comment]
@nlharris Can i contribute to this org. in any way?

**Comment by nlharris** [comment]
@akshit5565 the best way to help us is to follow us on social media and help us spread the word about BOSC 2023!
Website: https://www.open-bio.org/events/bosc/
Twitter:[ @OBF_BOSC](https://twitter.com/OBF_BOSC), #BOSC2023
Mastodon: https://genomic.social/@BOSC

**Comment by akshit5565** [comment]
@nlharris I want to contribute keeping in mind the GSOC Selection
Can you help me in that?

**Comment by akshit5565** [comment]
@nlharris ??
